packages/subgraph/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "pool-matadata-registry-graph",
```

Comment:
```suggestion
  "name": "pool-metadata-registry-subgraph",
```

Reply:
Should we do it as in contracts/package.json?

```
  "name": "@balancer-pool-metadata/subgraph",
  "version": "1.0.0",
```